sources/zafira-models/src/main/java/com/qaprosoft/zafira/models/db/Test.java

```diff
@@ -57,22 +57,18 @@
     private Set<Tag> tags;
 
     public Test() {
-        testConfig = new TestConfig();
+        this.testConfig = new TestConfig();
     }
 
     public String getNotNullTestGroup() {
         return testGroup == null ? "n/a" : testGroup;
     }
 
     public WorkItem getWorkItemByType(WorkItem.Type type) {
```

Comment:
For future changes let's apply the following rule of thumb: if you see a method that return null you check if it can deal with null value right away - analyze call chain, and see if something can be done immediately (e.g. thrown an exception if it is appropriate) OR wrap it with Optional to force API clients to handle null check explicitly. This method is actually a great example of second approach: we are iterating over a collection to find an item matching a predicate which is not guaranteed to be there - optional is something that feel native here by design.

Reply:
:ok_hand::skin-tone-3: